pkg/cli/add_vcluster_helm.go

```diff
@@ -24,31 +27,79 @@ type AddVClusterOptions struct {
 	AccessKey                string
 	Host                     string
 	CertificateAuthorityData []byte
+	All                      bool
 }
 
 func AddVClusterHelm(
 	ctx context.Context,
 	options *AddVClusterOptions,
 	globalFlags *flags.GlobalFlags,
-	vClusterName string,
+	args []string,
 	log log.Logger,
 ) error {
-	// check if vCluster exists
-	vCluster, err := find.GetVCluster(ctx, globalFlags.Context, vClusterName, globalFlags.Namespace, log)
-	if err != nil {
-		return err
+	var vClusters []find.VCluster
+	if len(args) == 0 && !options.All {
+		return errors.New("empty vCluster name but no --all flag set, please either set vCluster name to add one cluster or set --all flag to add all of them")
+	}
+	if options.All {
+		log.Debugf("add vcluster called with --all flag")
```

Comment:
is that really useful outside of development context?

Reply:
good catch - removed